doc/source/notebooks/advanced/coregionalisation.ipynb

```diff
@@ -64,7 +64,7 @@
KkD8djmxfkuktz+pyErAGLqiTlEVd4vStl2HCPf0sebpt9jU/A5Ny+fzzBuHfTcCd0fVqTXXTygPeNnx19afzrX1pyfNSADMrg3x0E0LALwlhok29T0WGmGLiGQosZjK9lsXsf3WRZx9cjUdXb28czTiPf7r2y5mVm2Vt8XL9VDru3zv8dd5uPVdakJBZ+T5kx2sefot9h4JU10xiXNOnVz0pTkTR9UATcvnex9O3A8zQ2XHh4IB7r3ufGqrK7wlhjuXnKtgnQGNsEVEMpS6HxvwRs2Jj7eHezAYgKSR+COt7zGrtsqbFgZwY7oBwj19/Kd1LxOztmhKc6Zbn06d6u63dtCHE/fYdDMSqzfv9AK61qkzpxG2iMgouPuxXW7ASXzcrYyWOhLf397Fl+pPH7TWDeCGu5i1XmnO1HXvfBtqfXrN02+xdP50bw3a3daWui79UOu7aWcktFVrbDTCFhHJseFG4uGePiK9/cN+fyalOYfKzM7laDVxfXrx2h0A3vp0pCdGf8KougxYfuEZXH/RTG9durpiEncuOTftddCoevQUsEVExkFqQKqtrqBx3jS+dO/vsn7txC5X7nYpN0ime++xSreNrSYUZPG5p/Kz5/cBzqi6pzdGpLefnz2/j6qKwLBBWVPgY5fVlLgx5kvGmNeNMf3GmLQbvePHXW6MecsYs88Y891s3lNExK+27TrEgfYIM6ZWYszQx41UmrNx3jTqqoPeyHfx2h3sPRKmrjqYl+lzS/J6dUV5IOm+gvL4yHYN+4/AXwE7hjrAGBMAmoArgLnAMmPM3CzfV0TEd1YsmMmdS87l6r+YTmqOlluPPGDMiKU5t+06RFs46rX37OhybreFozldG05NGnPXp//l9cOsvuRMplSV0xnppTPSy5Sqcm6//Bxu/txZOXt/SZbVlLi19g0AM9xHRbgA2Get3R8/9lfAEmB3Nu8tIuJHjfOmeduhplSVE+3rpysa42NV5Zx/xhRuv/zjXmnOa+tPTzu13DhvGve/dHBQe8/ZtaGcjrATt7GlTr0vnT+dsoTf/WXGcG396Tl7bxksH2vY04HE6vfvARemO9AYswpYBTBjxozxPzMRkTwbLgh+9aJZzDllckalOVOnpYd6LBtDJc+5pVZTt2stW9ei7mTjaMSAbYx5Bjg1zVN/a619PJcnY61dB6wDp/lHLl9bRKQYDJdBnum6r7sW7k6fgzOVfqA9kvMM7HTJc9UVk4b80KEM8PEzYsC21l6W5Xu8DyTOk5wWf0xEZEJKFwRHE+Qa503jp8/to+1YT9IIt25yRV6SznLxoUNGLx+FU14FzjbGzDLGBIGvAE/k4X1FRErStl2HaDvWM6ggSduxnrwVJBmqgIyMn6zWsI0xS4GfAnXAk8aYf7XWLjbG/D/ABmtto7W2zxjzDWA7EAB+Ya19PeszFxGZoDTCnZiMTVP/tRjU19fb1tbWQp+GiIhI3hhjXrPWpq1rolriIiIiPqCALSIi4gMK2CIiIj6ggC0iIuIDCtgiIiI+ULRZ4saYNuCdHL5kLdCew9ebqHQds6drmBu6jrmh65i9XF7DM6y1demeKNqAnWvGmNahUuUlc7qO2dM1zA1dx9zQdcxevq6hpsRFRER8QAFbRETEByZSwF5X6BMoEbqO2dM1zA1dx9zQdcxeXq7hhFnDFhER8bOJNMIWERHxLQVsERERHyi5gG2MudwY85YxZp8x5rtpnq8wxjwYf/5lY8zM/J9l8cvgOt5mjNltjPmDMeZZY8wZhTjPYjbSNUw47gvGGGuM0daaNDK5jsaYa+M/j68bY/4p3+foBxn8n55hjHneGPP7+P/rxkKcZzEzxvzCGHPEGPPHIZ43xpi749f4D8aY+Tk9AWttyXzh9Nt+G5gNBIF/A+amHHMzcG/89leABwt93sX2leF1vASoit/+G13H0V/D+HGTgR1AC1Bf6PMutq8MfxbPBn4PTInfP7nQ511sXxlex3XA38RvzwUOFvq8i+0LWATMB/44xPONwFOAARqAl3P5/qU2wr4A2Get3W+tjQK/ApakHLMEeCB++xHgUmOMyeM5+sGI19Fa+7y1NhK/2wKcludzLHaZ/CwC/D3wD8DxfJ6cj2RyHW8Amqy1nQDW2iN5Pkc/yOQ6WuDE+O2TgH/P4/n5grV2B3B0mEOWAButowX4mDFmWq7ev9QC9nTg3YT778UfS3uMtbYP+BCoycvZ+Ucm1zHR13A+VcqAEa9hfLrsdGvtk/k8MZ/J5GdxDjDHGPOSMabFGHN53s7OPzK5jn8H/LUx5j1gG/Bf8nNqJWW0vztHZVKuXkgmJmPMXwP1wMWFPhc/McaUAT8Gri/wqZSCSTjT4p/DmenZYYyZZ639c0HPyn+WAfdba//RGLMA2GSM+aS1tr/QJyaOUhthvw+cnnD/tPhjaY8xxkzCmfrpyMvZ+Ucm1xFjzGXA3wJXW2t78nRufjHSNZwMfBL4jTHmIM561xNKPBskk5/F94AnrLW91toDwB6cAC4DMrmOXwMeArDWNgMn4DS1kMxl9LtzrEotYL8KnG2MmWWMCeIklT2RcswTwMr47S8Cz9l4toB4RryOxpjzgPtwgrXWDAcb9hpaaz+01tZaa2daa2fi5AFcba1tLczpFq1M/k8/hjO6xhhTizNFvj+fJ+kDmVzHPwGXAhhjPoETsNvyepb+9wSwIp4t3gB8aK09lKsXL6kpcWttnzHmG8B2nKzIX1hrXzfG3Am0WmufAH6OM9WzDyd54CuFO+PilOF1/F9ANfBwPGfvT9baqwt20kUmw2soI8jwOm4H/tIYsxuIAd+x1mrWLEGG1/H/A9YbY27FSUC7XoOZZMaYLTgfDmvja/13AOUA1tp7cdb+G4F9QAT4ak7fX/8eIiIixa/UpsRFRERKkgK2iIiIDyhgi4iI+IACtoiIiA8oYIuIiPiAAraIiIgPKGCLiIj4wP8F/P8gu3wsYUMAAAAASUVORK5CYII=\n",
```

Comment:
how do you review a file like this? (I realise this is what you're trying desperately, and quite rightly, to move away from). For now, what should I be looking at?

Reply:
You don't - you check out the PR branch, remove the notebook .ipynb file, run `make` in the doc/source/notebooks directory, and check that it doesn't actually introduce any changes to the .ipynb file: this ensures that the .ipynb is up to date with the percent-py jupytext representation. (It requires the notebook to be deterministic, which if it's not already the case the PR author should add to an edited notebook)

Reply:
wow, just, wow. I feel like I've been taken back to the days of ad hoc tests that are never committed to version control